docs(sandbox): fix components with Portals inside sandbox (#2961)
kiwicom/orbit: packages/orbit-components/src/Dialog/README.md

| Name              | Type                    | Description                                                                                          |
| :---------------- | :---------------------- | :--------------------------------------------------------------------------------------------------- |
| dataTest          | `string`                | Optional prop for testing purposes.                                                                  |

> | insidePortal      | `boolean`               | Optional prop, if set to `false` component is not rendered inside Portal, default is `true`          |

Reviewer: I recommend telling people why they would want to turn this off, instead of explaining what it does. Let me know what you think.

```suggestion
| insidePortal      | `boolean`               | Optional prop, set it to `false` if you're rendering Dialog inside a custom portal, defaults to `true`          |
```

Same in readmes of other components, but without "optional prop" and "defaults to" parts.
